rear-panel/firmware/CMakeLists.txt

```diff
@@ -229,12 +229,22 @@ macro(rearpanel_loop)
       COMMENT "Flashing board"
       DEPENDS ${HEX_IMG_NAME})
   list(APPEND ${PROJECT_NAME}-IMAGES ${REVISION_TARGET}-image-hex)
-  list(APPEND ${PROJECT_NAME}-APPLICATIONS ${REVISION_TARGET}-hex)
+  list(APPEND ${PROJECT_NAME}-APPLICATIONS ${REVISION_TARGET}-bin)
   install_if_latest_revision(
     FILENAME ${HEX_IMG_FILENAME}
     COMPONENT robot-images
     THISREVISION ${REVISION}
     REVISIONS ${REVISIONS})
+  install_if_latest_revision(
```

Comment:
we want to install _all_ revisions here - we only do `install_if_latest_revision` for `robot-images` because `robot-images` builds the zip we'll send to the factory. this can be 
```
install(
      FILES ${CMAKE_CURRENT_BINARY_DIR}/${REVISION_TARGET}.bin
      DESTINATION applications/
      COMPONENT Applications)
```